pkg/ledger/executor_test.go

```diff
@@ -275,3 +343,52 @@ func TestMetadata(t *testing.T) {
 		assertBalance(t, l, "platform", "COIN", 15)
 	})
 }
+
+func TestSetTxMeta(t *testing.T) {
+	with(func(l *Ledger) {
+		defer l.Close(context.Background())
```

Comment:
Actually the 'with' method start an fx container which properly close the ledger at the end of the test, so it is useless to explicitely close it. But this is a no op and there is probably other tests with that.